### DISCUSSION TITLE
How to import existing data to Vercel Postgres Database

### DISCUSSION THREAD

**Discussion by st3fus** [vercel/community#2916] [Help] [answered]
### Summary

So I migrated my project, both FE and BE to Vercel and now I'd like to migrate my Postgre database. I already have the data dumped with pg_dump but I can't seem to restore it. Also, I've tried connecting to it with DBeaver but no success either... Anyone has a solution?

### Steps to Reproduce

pg_restore returns: error: could not execute query: ERROR:  role "postgres" does not exist (even if I change the role to "default" in the command)

**Comment by smaeda-ks** [comment, marked as answer]
@st3fus depends on how you set up your Vercel Storage, you should be able to identify your role name:
- https://neon.tech/docs/guides/vercel
- https://neon.tech/docs/guides/vercel-manual

Did you specify the right role name that you configured in those steps?

**Reply by st3fus** [reply]
My Vercel role is named 'default' and on my AWS Postgres Database it's "postgres"

**Reply by smaeda-ks** [reply]
@st3fus This section contains some notes:
https://neon.tech/docs/import/import-from-postgres#data-import-notes

> Because pg_dump dumps a single database, it does not include information about roles stored in the global pg_authid catalog. Also, Neon does not support creating roles using psql. You can only create roles using the Neon Console. If you do not create roles in Neon before importing a database that has roles, you will receive "role does not exist" errors during the import operation. You can ignore these errors if they occur. They do not prevent data from being imported.

Also: https://neon.tech/docs/manage/roles
